### ISSUE TITLE
Reshade box around box stats on viewer after death to signal that bot is dead

### ISSUE THREAD

**Issue by JamesDarby345** [dbakewel/netbots#100]
**Is your feature request related to a problem? Please describe.**
I am frustrated when I cant tell who is dead really fast
**Describe the solution you'd like**
change the colour of the box after the bot dies, reset it aftera new game starts
**Describe alternatives you've considered**

**Additional context**

**Comment by dbakewel** [comment]
Yes, that's a very cool idea. We could use the color variant function and just reduce the value of the color by 150. You can see how to do this for the current direction arrows.

**Comment by mcstoer** [comment]
Would we have to store the individual bot's colour as a separate variable or can we get the colour from the tkinter widgets?
Changing the colour would be done under the `if bot['health'] == 0:` portion and we can check for a new game to reset the colours, but what happens if instant replay is activated?  Where would we revert the colours?
I can work on this, but at the moment I'm unsure how to apply these changes.
